zeno/classes/report.py

```diff
@@ -22,9 +22,11 @@ class Parameters(CamelModel):
 
 
 class Report(CamelModel):
+    id: int
```

Comment:
Do we need an ID? Or can we use the name as the ID and ensure it's always unique like with slices?